src/Command/ExecuteCoreUpgradeScriptsCommand.php

```diff
@@ -108,6 +108,8 @@ protected function execute(InputInterface $input, OutputInterface $output){
 				$fsHelper->move($oldSourcesDir . '/__apps', $oldSourcesDir . '/apps');
 				$output->writeln($plain);
 			} catch (ProcessFailedException $e){
+				$fsHelper->removeIfExists($oldSourcesDir . '/apps');
+				$fsHelper->move($oldSourcesDir . '/__apps', $oldSourcesDir . '/apps');
```

Comment:
@Phiber2000 It will be executed twice in case move in the line 108 fails which is nearly impossible. So I think this is ok.